account-integrations/safe/src/lib/FCL_Webauthn.sol

```diff
@@ -135,4 +132,4 @@ library FCL_WebAuthn {
 
         return result;
     }
-}
+}
```

Comment:
I assume this file is now back to it's original version?

Reply:
Not quite, I raised https://github.com/getwax/wax/pull/93 to revert more of the libs back to their original state. Remaining work to restore to it's original state is covered by #84 & #94